opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -498,27 +522,18 @@ namespace Opm
             return;
         }
 
-        const bool old_well_operable = this->operability_status_.isOperable();
-
         updateWellOperability(ebos_simulator, well_state, deferred_logger);
```

Comment:
In the old code, updateWellOperability() gets called before you get the well_operable variable. In the new code, this is not done. Would that not change behaviour? Is the call unnecessary now?

Reply:
The logic should be the same. The call to get the operability status is moved outside the checkWellOperability function. But maybe I misunderstood your comment. If there is a difference it is not intended.

Reply:
Before:
```
const bool old_well_operable = this->operability_status_.isOperable();
updateWellOperability(ebos_simulator, well_state, deferred_logger);
const bool well_operable = this->operability_status_.isOperable();
// Logic dealing with changes in operability
```

Now:
```
const bool old_well_operable = this->operability_status_.isOperable();
// New code that may update the solvable_ flag.
const bool well_operable = this->operability_status_.isOperable();
// Logic dealing with changes in operability
updateWellOperability(ebos_simulator, well_state, deferred_logger);
```

Reply:
Now (according to my intention and how I still read the code):

```
const bool old_well_operable = this->operability_status_.isOperable();
// Logic dealing with changes in operability
checkWellOperability(ebos_simulator, well_state, deferred_logger);
updateWellOperability(ebos_simulator, well_state, deferred_logger);  (called from within checkWellOperability)
// New code that may update the solvable_ flag.
const bool well_operable = this->operability_status_.isOperable();
```

Reply:
Ok, I see now that I misunderstood the change.